website/docs/reference/stickiness.md

```diff
@@ -25,11 +25,21 @@ Custom stickiness is available starting from Unleash Enterprise v4.
 When using [the gradual rollout strategy](../reference/activation-strategies.md#gradual-rollout) or [feature toggle variants](./feature-toggle-variants.md), you can use parameters other than the user id to calculate stickiness. More specifically, you can use any field, custom or otherwise, of the [Unleash Context](../reference/unleash-context.md) as long as you have enabled custom stickiness for these fields.
 
 :::note
-This feature is currently in beta and is not yet supported by all our SDKs. Check out the [SDK compatibility table](../reference/sdks/index.md#server-side-sdk-compatibility-table) to see what SDKs support it at the moment.
+This feature is currently generally available but is not yet supported by all our SDKs. Check out the [SDK compatibility table](../reference/sdks/index.md#server-side-sdk-compatibility-table) to see what SDKs support it at the moment.
```

Comment:
I suggest replacing this line and surrounding `:::` lines with this:

```suggestion
:::note SDK compatibility

Custom stickiness is supported by all of our SDKs except for the Rust SDK. You can always refer to the [SDK compatibility table](../reference/sdks/index.md#server-side-sdk-compatibility-table) for the full overview.

:::
```

The extra spacing is just for formatting reasons. The extra text after `note` gives the admonition a little header. It's only the Rust SDK that doesn't support it, so I think we could call that out explicitly to save people finding it in the table.

Reply:
Sounds good